### DISCUSSION TITLE
What is the best time you can get with only inf jump

### DISCUSSION THREAD

**Discussion by nekoify** [aeolus-1/triangleGame.github.io#52] [General]
this is my best time
<img width="1440" alt="Screen Shot 2022-04-22 at 9 20 37 am" src="https://user-images.githubusercontent.com/98920372/164566641-b76e1870-4bdf-4e75-a072-3d7f1c6ba526.png">

**Comment by serifonium** [comment]
https://speedrun.com

**Comment by Bruce-Browninggg** [comment]
you dumbass jake none of us have infin jump becuase ben deleted it
